packages/lab/src/layout/LayerLayout/LayerLayout.tsx

```diff
@@ -0,0 +1,111 @@
+import { forwardRef, HTMLAttributes, useState, useEffect } from "react";
+import cx from "classnames";
+
+import { makePrefixer, Breakpoints } from "@jpmorganchase/uitk-core";
+import { Scrim, ScrimProps } from "../../scrim";
+import "./LayerLayout.css";
+import { useChangeView } from "../utils";
+
+export const LAYER_POSITION = [
+  "center",
+  "left",
+  "top",
+  "right",
+  "bottom",
+] as const;
+
+export type LayerPosition = typeof LAYER_POSITION[number];
+
+export interface LayerLayoutProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Display or hide the scrim.
+   */
+  displayScrim?: boolean;
```

Comment:
I'm wondering if it would be better to remove scrim from this component and get the consumer to add it via composition if they need it?

This would avoid passing `displayScrim` and `scrimProps` to this component which I think we were trying to avoid.

@heswell / @joshwooding what do you think?

Reply:
@heswell @joshwooding @origami-z any thoughts on this please? In the spec we have "scrim displayed by default" hence I've added it as part of the component